app/lib/oi/oi.go

```diff
@@ -0,0 +1,102 @@
+package oi
```

Comment:
We might need a better name for this package: output (?) or as we just discussed merge it with package `dbg`